vere/http.c

```diff
@@ -2600,17 +2555,19 @@ _proxy_ward_plan(u3_ward* rev_u)
     rev_u->non_u = uv_buf_init((c3_c*)non_w, len_w);
   }
 
-  // TODO: all this forwarding stuff doesn't work with %light/http-server.
-  
-  // XX confirm duct
-  u3_noun pax = u3nq(u3_blip, c3__http, c3__prox,
-                     u3nc(u3k(u3A->sen), u3_nul));
+  {
+    u3_noun who = u3i_chubs(2, rev_u->who_d);
+    u3_noun cha = u3nq(c3__a, c3__give, c3__prox, u3_nul);
+    u3_noun dat = u3nc(c3__that, u3nt(rev_u->por_s,
+                                      u3k(rev_u->con_u->sec),
+                                      non));
 
-  u3_noun wis = u3nc(c3__wise, u3nq(u3i_chubs(2, rev_u->who_d),
-                                    rev_u->por_s,
-                                    u3k(rev_u->con_u->sec),
-                                    non));
-  u3v_plan(pax, wis);
+    cad = u3nq(c3__want, who, cha, dat);
+  }
+
+  //  XX s/b c3__ames?
```

Comment:
What's s/b ?

Reply:
"should be". All the unix/ames ducts use `c3__newt`.